<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ALDOTDemo</class>
 <widget class="QMainWindow" name="ALDOTDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALDOT Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_up">
      <property name="toolTip">
       <string>Go up, way up</string>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn_down">
      <property name="toolTip">
       <string>Go down, way down</string>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cb_ice">
        <property name="text">
         <string>Ice</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_lemon">
        <property name="text">
         <string>Lemon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_sugar">
        <property name="text">
         <string>Sugar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>590</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>btn_up</tabstop>
  <tabstop>btn_down</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>cb_ice</tabstop>
  <tabstop>cb_lemon</tabstop>
  <tabstop>cb_sugar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>ALDOTDemo</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
